aspnetcore/fundamentals/routing/samples/6.0/RoutingSample/Routing/ICoolMetadata.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace RoutingSample.Routing
+{
```

Comment:
If there are 6.0 apps, we could use file scoped namespaces: 

```suggestion
namespace RoutingSample.Routing;
```

Reply:
If I use Add Class from VS, for example, it generates the old-style namespace blocks. It looks like some of the dotnet new templates use the newer approach, so I'm ending up with a bit of a mixture. Is it worth me going through the code once I'm done and just switching it all over to the newer version?

Reply:
FYI @DamianEdwards.

Reply:
Let's go with file scoped namespaces. VS will catch up someday. It's much cleaner and helps with the indent problem with snippets.